seacrowd/sea_datasets/lti_langid_corpus/lti_langid_corpus.py

```diff
@@ -0,0 +1,310 @@
+class LTILangIDDataset(datasets.GeneratorBasedBuilder):
+    """LTI LangID corpus is a dataset for language identification for 1266 languages."""
+
+    BUILDER_CONFIGS = (
+        [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_source",
+                version=datasets.Version(_SOURCE_VERSION),
+                description=f"{_DATASETNAME} source schema",
+                schema="source",
+                subset_id=f"{_DATASETNAME}_ifa",
+            ),
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_seacrowd_text",
+                version=datasets.Version(_SEACROWD_VERSION),
+                description=f"{_DATASETNAME} SEACrowd schema",
+                schema="seacrowd_text",
+                subset_id=f"{_DATASETNAME}_ifa",
+            ),
+        ]
+        + [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_{LANG}_source",
+                version=datasets.Version(_SOURCE_VERSION),
+                description=f"{_DATASETNAME} source schema",
+                schema="source",
+                subset_id=f"{_DATASETNAME}_{LANG}",
+            )
+            for LANG in _LANGUAGES
+        ]
+        + [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_{LANG}_seacrowd_text",
+                version=datasets.Version(_SEACROWD_VERSION),
+                description=f"{_DATASETNAME} SEACrowd schema",
+                schema="seacrowd_text",
+                subset_id=f"{_DATASETNAME}_{LANG}",
+            )
+            for LANG in _LANGUAGES
+        ]
+    )
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+
+            features = datasets.Features({"text": datasets.Value("string"), "language": datasets.Value("string")})
+
+        elif self.config.schema == "seacrowd_text":
+            features = schemas.text_features(label_names=_LANGUAGES)
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        data_dir = dl_manager.download_and_extract(_URL)
+        data_dir = os.path.join(data_dir, "MIL-TALE", "5")
+
+        lang_id = self.config.subset_id.split("_")[-1]
+
+        datasplits_dir = os.path.join(data_dir, "datasplits")
+
+        if not Path(datasplits_dir).exists() or not Path(os.path.join(datasplits_dir, "DONE")).exists():
+            # Run provided install.sh to generate train/dev/test splits in "datasplits" folder
+            subprocess.call([os.path.join(data_dir, "code", "install.sh"), datasplits_dir])
+            with open(os.path.join(datasplits_dir, "DONE"), "w"):
+                pass
```

Comment:
My current implementation is a hack that uses subprocess.call to execute install.sh from the dataset's cache folder but this would not work for people who are not running this from a bash environment. Unfortunately, I no longer have the bandwidth to convert this to something with better cross-platform compatibility. 
I can release this dataset if someone else wants to give it a try. @holylovenia

Reply:
I think it's fine for now. This hack will be helpful for the time being. Thanks for having thought of it. 😁 

Could you please add information regarding "uses subprocess.call to execute install.sh from the dataset's cache folder but this would not work for people who are not running this from a bash environment" in the `_DESCRIPTION`?